qiskit_dynamics/dispatch/wrap.py

```diff
@@ -125,3 +69,150 @@ def wrapped_function(*args, **kwargs):
         return result
 
     return wrapped_function
+
+
+def _wrap_args(args):
+    """Return wrapped args"""
+    return tuple(_wrap_array_function(x) if isinstance(x, FunctionType) else x for x in args)
```

Comment:
Should this do: `... else Array._wrap(x) ...`?

Reply:
I guess I have this backwards, it's supposed to remove `Array` wrapping here right?

Reply:
Ah sorry I realize now how the `__qiskit_array__` method is being used now at the beginning after reading the docs update. Disregard this comment!